lib/utils.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs')
+const constants = require('./constants')
+
+/**
+ * Write a config file to disk
+ *
+ * @param {string} platform - platform to write
+ * @param {object} config - the object to write
+ */
+function writeConfigToDisk (platform, config, v1Config) {
+    fs.writeFileSync(`${constants.GENERATED_DIR}/v2/${platform}-config.json`, JSON.stringify(config, null, 4))
+    fs.writeFileSync(`${constants.GENERATED_DIR}/v1/${platform}-config.json`, JSON.stringify(v1Config, null, 4))
```

Comment:
This v1 saving would be better as part of the legacy generation, in hindsight the `v1Config` argument here seems like a mistake...